spec/integration/puppet_x/puppetlabs/powershell_manager_spec.rb

```diff
@@ -58,6 +58,18 @@ def create_manager
         expect(first_pid).to eq(second_pid)
       end
 
+      it "should fail if the manger is created with a short timeout" do
+        expect {
+          manager = PuppetX::PowerShell::PowerShellManager.new(manager_args, false, 0.01)
```

Comment:
Don't need to assign to `manager` here

Reply:
I think we might want to add at least a little note about the magic of why this works... i.e. anything less than the `sleep_interval` for the manager results in 0 iterations / no waiting:

```
[1] pry(main)> ( 0.01 / 0.2)
=> 0.049999999999999996
[2] pry(main)> ( 0.01 / 0.2).to_int
=> 0
[3] pry(main)> ( 0.01 / 0.2).to_i
=> 0
[4] pry(main)> ( 0.1 / 0.2).to_i
=> 0
[5] pry(main)> ( 0.15 / 0.2).to_i
=> 0
[6] pry(main)> ( 0.2 / 0.2).to_i
=> 1
```

Reply:
Fixed in the latest push.